Source/ablastr/utils/text/StreamUtils.H

```diff
@@ -0,0 +1,24 @@
+/* Copyright 2023
+ *
+ * This file is part of WarpX.
+ *
+ * License: BSD-3-Clause-LBNL
+ */
+
+#ifndef ABLASTR_UTILS_TEXT_STREAMUTILS_H_
+#define ABLASTR_UTILS_TEXT_STREAMUTILS_H_
+
+#include <iostream>
```

Comment:
That is a lighter include to pull in:
```suggestion
#include <istream>
```